WordPress/openverse-frontend
# New 404 page

## Fixes


Fixes #256 by @zackkrida 

## Description



Implements the new 404 page, with a search bar for an 'all results' search. Sadly this needs to be based on the VHeader branch since there's searchbar functionality fixes there.

<img width="381" alt="CleanShot 2022-01-06 at 15 25 01@2x" src="https://user-images.githubusercontent.com/6351754/148456750-395df211-9581-4752-94db-01b94d6c3e59.png">

<img width="1443" alt="CleanShot 2022-01-06 at 15 24 47@2x" src="https://user-images.githubusercontent.com/6351754/148456760-efb0c0a4-36fa-470a-9d45-1dae573b5717.png">



## Testing Instructions


Visit any fake url, like http://localhost:8443/dfsgfefad, and do some testing:

- Observe the page on different viewport sizes
- Submit a search
- Try the 'home' link

## Checklist


[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin


<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
